spec/aep-63/README.md

```diff
@@ -0,0 +1,131 @@
+
+#### Certificate Management
+
+While there will be a documented API for this - it will likely be handled automagically under the hood for the managed wallet user. If it is handled for the user we may not implement the list and revoke endpoints
+
+`POST /v1/certificates` (create cert)
+`GET /v1/certificates` (list all certs)
+`DELETE /v1/certificates/{id}` (revoke cert)
+
+#### Deployment Creation
+
+`POST /v1/deployments` (create deployment based on `SDL/ YAML` and escrow `deposit`)
+
+Implementation tracked in [issue #767](https://github.com/akash-network/console/issues/767)
+
+#### Bid Selection
+
+`GET v1/bids/{dseq}` (get bids for a `dseq`)
+
+Implementation tracked in [issue #767](https://github.com/akash-network/console/issues/767)
+
+#### Lease Creation
+
+`POST /v1/leases` (create lease based on a payload that is an array of sdl, dseq, gseq, provider combinations)
+
+Note that Akash supports creating multiple leases from a single SDL - where each lease can be for a different service (with a different container image) and deployed to a different provider
+
+Implementation tracked in [issue #767](https://github.com/akash-network/console/issues/767)
+
+#### Listing Deployments
+
+`GET /v1/deployments` (list all deployments)
+`GET /v1/deployments/{dseq}` (list deployment detail for a specific deployment)
+
+Implementation tracked in [issue #767](https://github.com/akash-network/console/issues/767)
+
+#### Deployment Closure
+
+`DELETE /v1/deployments/{dseq}` (close a specific deployment)
+
+Implementation tracked in [issue #767](https://github.com/akash-network/console/issues/767)
+
+#### Funding Deployments
+
+For viewing or adding funds to the escrow of a specific deployment
+
+`GET /v1/deployments/deposit{dseq}` (retrieve escrow balance details)
+`POST /v1/deployments/deposit/{dseq}` (accepts `deposit` amount in the body)
+
+implementation tracked in [issue #989](https://github.com/akash-network/console/issues/989) and [issue #990](https://github.com/akash-network/console/issues/990)
+
+#### Funding Account
+
+This is for the customer to purchase more credits and fund their account - we will decide if we offer this or not based on customer requests but it will likely directly go to the Stripe Checkout API https://docs.stripe.com/api/checkout/sessions/object
+
+#### Retrieving Logs
+
+`GET/v1/logs/{dseq}`
+
+#### Retrieving Events
+
+`GET/v1/events/{dseq}`
+
+### UI for Managing API Keys
+
+The user will be able to get to an API key management page from their user profile drop down
+
+![api-keys dropdown menu](api-keys-menu.png)
+
+The API key management page will let the user view all keys, create a new one or delete and existing one
+
+![api-keys management page](api-key-management.png)
+
+### Documentation
+
+Documentation will be added in three places
+
+- In the [Swagger docs](https://console-api.akash.network/v1/swagger) for Console API (which is linked from the Console side nav bar)
```

Comment:
We don't have user routes in the swagger docs currently

Reply:
but it is something we can add?

Reply:
We can certainly add a separate swagger doc for managed wallet api

Reply:
https://github.com/akash-network/console/issues/1043